kubernetes/nodejs-ingress/waypoint.hcl

```diff
@@ -0,0 +1,34 @@
+project = "example-nodejs"
```

Comment:
Can we make the project and app names the same as the folder? Otherwise if you try running multiple examples, they name collide.

Reply:
Ah yes, oops! Good call 👍🏻